modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/import/new_import/import_layouts_resources.jsp

```diff
@@ -431,6 +431,15 @@ ManifestSummary manifestSummary = ExportImportHelperUtil.getManifestSummary(user
 					</c:when>
 					<c:otherwise>
 						<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" cssClass="options-group" label="update-data">
+							<c:if test="<%= !group.isCompany() %>">
```

Comment:
Is this the same as `stagingGroupHelper.isCompanyGroup(group)` ?

Reply:
Thanks @jaimelr10 , good catch! ❤️ 


I prefer `stagingGroupHelper.isCompanyGroup(group)` too, updated it! 🙌